apps/root/src/services/labelService.ts

```diff
@@ -116,22 +98,4 @@ export default class LabelService implements ILabelService {
       console.warn(`Label for address ${labeledAddress} does not exist.`);
     }
   }
-
-  setWalletsAliases(): void {
-    this.accountService.setWalletsAliases(this.labels, this.ens);
-  }
-
-  async initializeAliasesAndContacts(): Promise<void> {
-    const labelsAndContactList = await this.fetchLabelsAndContactList();
-    if (labelsAndContactList) {
-      this.labels = labelsAndContactList.labels;
-      this.contactListService.setContacts(labelsAndContactList.contacts);
```

Comment:
Still not sure why we moved this to contacts

Reply:
I found the following use case:
User has a labeled wallet connected, and then wants to add it as a contact, and when it does, chooses a new label. That label should be updated in labelService, so it's synced with both contactList and accountService's wallets.
Adding, editing or removing a label never affects the contactList. But addContact contactList method has an effect on labels.